x-pack/qa/repository-old-versions/src/test/java/org/elasticsearch/oldrepos/OldMappingsIT.java

```diff
@@ -143,6 +160,18 @@ public void setupIndex() throws IOException {
             doc3.setJsonEntity(Strings.toString(bodyDoc3));
             assertOK(oldEs.performRequest(doc3));
 
+            Request doc4;
+            if (oldVersion.before(Version.fromString("6.0.0"))) {
+                doc4 = new Request("POST", "/" + "standard_token_filter" + "/" + "my_type");
+
+            } else {
+                doc4 = new Request("POST", "/" + "standard_token_filter" + "/" + "doc");
```

Comment:
do we need different types here? If I recall correctly v5/v6 still need a type but in v5 we could also use "doc"?